core/src/com/unciv/logic/civilization/QuestManager.kt

```diff
@@ -37,6 +39,8 @@ class QuestManager {
 
         const val GLOBAL_QUEST_MAX_ACTIVE = 1
         const val INDIVIDUAL_QUEST_MAX_ACTIVE = 2
+
+        const val CONTEST_LENGTH = 30
```

Comment:
Not depended on game speed? I don't remember myself, so feel free to ignore this is if it isn't

Reply:
It is depending on game speed, that's just an unused constant. I'll delete it.